src/cli/cli_mdns.cpp

```diff
@@ -879,6 +879,64 @@ void Mdns::HandleIp4AddressResult(otInstance *aInstance, const otMdnsAddressResu
     Interpreter::GetInterpreter().mMdns.HandleAddressResult(*aResult, kIp4Address);
 }
 
+template <> otError Mdns::Process<Cmd("recordquerier")>(Arg aArgs[])
```

Comment:
Did we need to document any of these MDNS CLI commands for the site?

Reply:
None of the CLI `mdns` commands are currently documented (as they are intended for testing only). This has been intentional. https://github.com/openthread/openthread/pull/9797#discussion_r1546791369

Later, we may change this (allow external use of native OT mDNS) and add documentation for them all (in its own PR).

When I get the chance, one of the things I want to do is add documentation along with many examples (a guide) for all CLI `mdns` commands, but that's a lower priority for now, and I prefer it to be done after all related mDNS features are completed (so we can have a consistent set of commands and examples).